src/mon/MonClient.cc

```diff
@@ -1154,9 +1167,12 @@ void MonClient::start_mon_command(const std::vector<string>& cmd,
                                   ceph::buffer::list *outbl, string *outs,
                                   Context *onfinish)
 {
+  ldout(cct,10) << __func__ << " cmd=" << cmd << dendl;
   std::lock_guard l(monc_lock);
   if (!initialized || stopping) {
-    onfinish->complete(-ECANCELED);
+    if (onfinish) {
```

Comment:
note for posterity, we do have callers who don't pass a valid onfinish to `MonClient::start_mon_command()`, for instance, see https://github.com/ceph/ceph/blob/master/src/mgr/DaemonServer.cc#L2628